README.md

```diff
@@ -1,72 +1,97 @@
+              :
+      TEAM_NAME_N: read | triage | write | maintain | admin
     # Optional repository settings
     settings:
       has_wiki: true|false # Default is false
     visibility: public | private # Default is public
 ```
 
-Note: the ```name: repo name``` does not necessarily appear as the first field in a repositories entry which can be confusing.
+> [!IMPORTANT]
+>
+> - Invite users to join the CNCF Org *before you add them via a PR* to config.yaml
+> - GitHub usernames are case sensitive, mixed case usernames *should match* how they appear in their GitHub profile page
+> - If you grant access to a repo by any other means (via the GitHub web app, updating settings.yml in a repository's .github directory) and the access granted is not already described in the config.yaml file here, then CLOWarden will revert the access granted outside of CLOWarden each time it checks the config.yaml file.
 
- Named teams referenced in a ```repository``` entry are also defined in config.yaml under ```teams```. (much further down the file, beyond line 10,000)
+> [!NOTE]
+>
+> - CLOWarden will check your Pull Request changes to config.yaml and report any errors before they are merged and The CNCF Projects Team are here to help.
+> - Having ```name: REPO_NAME``` appear as the first key in an repository array entry makes it easier to find and read the entry. Placing it elsewhere in the entry is valid YAML but if you want to make it easier to update we suggest you add the name field first.
+
+Named teams referenced in an entry under ```repositories``` are also defined in config.yaml under ```teams```. (much further down the file, beyond line 10,000)
 
 ```yaml
 teams:
-  - name: team_name
+  - name: TEAM_NAME
+    displayName: You can add a full team name here with spaces.
     maintainers:
-      - github_profile_1
+      - GITHUB_USERNAME_1
+               |
                |
-      - github_profile_n
+      - GITHUB_USERNAME_N
     members:
       has_wiki: true|false
-    displayName: Team name that can have spaces used to create a Slack Channel
-    slack: {true|false|Slack channel name}  # Create a Slack channel for this team
     secret: {true|false} # Hidden GitHub Team
 ```
 
-#### Notes
+## Notes
 
-[cncf/sheriff](https://github.com/cncf/sheriff) periodically reads config.yaml on the main branch to apply the permissions to CNCF orgs, so once your PR is approved, the [Sheriff Apply GitHub action](https://github.com/cncf/people/actions/workflows/apply.yml ) will run to apply your changes.
+After your PR is merged to main, CLOWarden will apply changes you have requested and then report the access changes made taken on [clowarden.io/audit](https://clowarden.io/audit/?page=1)
 
-[cncf/sheriff](https://github.com/cncf/sheriff) is a fork of [electron/sheriff](https://github.com/electron/sheriff). The cncf fork has code to cover CNCF-specific procedures. Thank you Electron Sheriff contributors.
+Changes are applied within an hour of being merged.
```

Comment:
We reference 10 mins earlier - should this be the same, or do we need to change the previous listing to 1 hr?